### PR TITLE
xcode-build-version 0.0.1

### PR DESCRIPTION
**Pull request by TiagoBrasN**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

Outputs the project's version and build number as environmental variables:
- **XBV_PROJECT_VERSION**
- **XBV_PROJECT_BUILD**

### CODE REVIEW

File: steps/xcode-build-version/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    is_required: false
+    summary: Path to the 'xcodeproj'
+    title: Path to the 'xcodeproj'
+- bitrise_tag_info_plist_path: ""
+  opts:
+    description: |
+      File of your release 'Info.plist' file.
+
+      If not set it tries to find it.
+    is_required: true
```
Comment (godrei) [inline steps/xcode-build-version/0.0.1/step.yml]: s_required: true, but description states: `If not set it tries to find it.`

File: steps/xcode-build-version/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- XBV_PROJECT_VERSION: null
+  opts:
+    description: |
+      Project Version, eg, 1.3.9
+    summary: Project version.
+    title: CFBundleShortVersionString
+- XBV_PROJECT_BUILD: null
+  opts:
+    description: |
+      Project vuild number, eg, 85
```
Comment (godrei) [inline steps/xcode-build-version/0.0.1/step.yml]: typo: Project **vuild** number

File: steps/xcode-build-version/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+title: Xcode Build Version
```
Comment (godrei) [inline steps/xcode-build-version/0.0.1/step.yml]: It seems this step is about the Xcode project's version and build number, I would suggest a better title, like: `Get Xcode project version`

File: steps/xcode-build-version/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (godrei) [inline steps/xcode-build-version/0.0.1/step.yml]: it seems the step is not using these dependencies, avid listing unused dependencies, as it increases the step run time.

File: steps/xcode-build-version/0.0.1/step.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- bitrise_tag_xcodeproj_path: ""
```
Comment (godrei) [inline steps/xcode-build-version/0.0.1/step.yml]: question: why not `bitrise_tag_xcodeproj_path` is the key of the input?

File: steps/xcode-build-version/0.0.1/step.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+website: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+source_code_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+support_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version/issues
+published_at: 2020-02-05T13:56:37.566709Z
+source:
+  git: https://github.com/nodes-ios/bitrise-step-xcode-build-version.git
+  commit: b8d43d6c81c4b5a5c32d03bf199f8e304ed88c1e
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```
Comment (koral--) [inline steps/xcode-build-version/0.0.1/step.yml]: Does it make sense to run this step on Linux?

File: steps/xcode-build-version/0.0.1/step.yml (hunk 2 above)
Comment (koral--) [inline steps/xcode-build-version/0.0.1/step.yml]: typo `eg,` -> `eg.`

File: steps/xcode-build-version/0.0.1/step.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
+title: Xcode Build Version
+summary: |
+  Gets the project's version and build number
```
Comment (koral--) [inline steps/xcode-build-version/0.0.1/step.yml]: I would use `version and build number of the project`.

File: steps/xcode-build-version/0.0.1/step.yml (hunk 2 above)
Comment (lszucs) [reply steps/xcode-build-version/0.0.1/step.yml]: typo: `vuild` should be `build`

**Comment by lpusok** [comment]
Hi @TiagoBrasN,
Thanks for the contribution!
Please send in a new PR with the fixes. Will close the current one.

**Pull request closed without merging**